code/modules/clothing/head/mind_monkey_helmet.dm

```diff
@@ -0,0 +1,106 @@
+/obj/item/clothing/head/monkey_sentience_helmet
+	name = "monkey mind magnification helmet"
+	desc = "A fragile, circuitry embedded helmet for boosting the intelligence of a monkey to a higher level. You see several warning labels..."
+
+	icon_state = "monkeymind"
+	strip_delay = 100
+	clothing_flags = EFFECT_HAT
+	COOLDOWN_DECLARE(message_cooldown) //It'll get annoying quick when someone tries to remove their own helmet 20 times a second
+	var/datum/mind/magnification = null ///A reference to the mind we govern
+
+/obj/item/clothing/head/monkey_sentience_helmet/examine(mob/user)
+	. = ..()
+	. += "<span class='boldwarning'>---WARNING: REMOVAL OF HELMET ON SUBJECT MAY LEAD TO:---</span>"
+	. += "<span class='warning'>BLOOD RAGE</span>"
+	. += "<span class='warning'>BRAIN DEATH</span>"
+	. += "<span class='warning'>PRIMAL GENE ACTIVATION</span>"
+	. += "<span class='warning'>GENETIC MAKEUP MASS SUSCEPTIBILITY</span>"
+	. += "<span class='boldnotice'>Ask your CMO if mind magnification is right for you.</span>"
+
+/obj/item/clothing/head/monkey_sentience_helmet/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_HEAD)
+		return
+	if(!ismonkey(user) || user.key)
+		to_chat(user, "<span class='boldnotice'>You feel a stabbing pain in the back of your head for a moment.</span>")
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+		if(isliving(user)) //I don't know what normally would force us to check this, but it's worth checking
+			var/mob/living/M = user
+			M.apply_damage(5,BRUTE,BODY_ZONE_HEAD,FALSE,FALSE,FALSE) //notably: no damage resist (it's in your helmet), no damage spread (it's in your helmet)
+			return
+		return
+	INVOKE_ASYNC(src, PROC_REF(poll), user)
+
+/obj/item/clothing/head/monkey_sentience_helmet/proc/poll(mob/living/carbon/monkey/user) //At this point, we can assume we're given a monkey, since this'll put them in the body anyways
+	user.visible_message("<span class='warning'>[src] powers up!</span>")
+	playsound(src, 'sound/machines/ping.ogg', 30, TRUE)
+	var/list/candidates = pollCandidatesForMob("Do you want to play as a mind magnified monkey?", ROLE_MONKEY_HELMET, null, ROLE_MONKEY_HELMET, 50, user, POLL_IGNORE_MONKEY_HELMET)
```

Comment:
```suggestion
	var/list/candidates = pollCandidatesForMob(
		Question = "Do you want to play as a mind magnified monkey?", 
		jobbanType = ROLE_MONKEY_HELMET, 
		gametypeCheck = null, 
		be_special_flag = null,
		poll_time = 100, 
		M = user, 
		ignore_category = POLL_IGNORE_MONKEY_HELMET)
```
to be honest, this proc is bad to read. Specified arguments will be better.
Also, `be_special_flag` can be null because we're going to make it always visible.